CMakeLists.txt

```diff
@@ -431,6 +438,8 @@ ELSE()
   MESSAGE("-- ALBANY_PARALELL_EXODUS set to False")
 ENDIF()
 
+SET(ALBANY_PARALELL_EXODUS FALSE)
```

Comment:
This line is setting `ALBANY_PARALELL_EXODUS` support to false in all cases. I think it should be removed.

Reply:
Good catch!  I had this in there initially for testing purposes and forgot to remove it!  I just checked in the corrected CMakeLists.txt file.